### DISCUSSION TITLE
프로퍼티 선언 순서가 고민된다구요? 이글 한번 참고해보세요!🧐

### DISCUSSION THREAD

**Discussion by compuTasha** [DeveloperAcademy-POSTECH/MacC-GetARock#52] [개발 팁]
이전 프로젝트에서 에이버리가 공유해준 프로퍼티 선언 순서 관련된 자료인데
좋아서 여기도 공유합니당

![](https://user-images.githubusercontent.com/63157395/180236841-4ab87396-da21-4a38-9427-358775c78116.png)

https://github.com/eure/swift-style-guide

**Comment by MARKGRAYSIRO** [comment]
진짜 좋은 정보네요..!!! 앞으로 어머니의 말씀처럼 가슴에 새겨넣고 개발하겠습니다.

**Reply by compuTasha** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 감사합니다..🙇🏻
